Kconfig.zephyr

```diff
@@ -1037,6 +1037,12 @@ config WARN_EXPERIMENTAL
 	  Print a warning when the Kconfig tree is parsed if any experimental
 	  features are enabled.
 
+config INSECURE
+	bool
+	help
+	  Symbol that must be selected by a feature if it is considered to be
```

Comment:
This feels a little awkwardly worded. Maybe: "Symbol to be selected by a feature to indicate that feature is insecure."